frontend/src/utils/TaskUtils.ts

```diff
@@ -44,94 +44,70 @@ export enum SortingTypes {
   SORT_TOTAL_WORK,
 }
 
-export const totalRatingsByDimension = (
-  task: Task,
-): Map<TaskRatingDimension, { sum: number; count: number }> => {
-  return task.ratings.reduce((result, { value, dimension }) => {
-    const { sum, count } = result.get(dimension) || { sum: 0, count: 0 };
-    return result.set(dimension, { sum: sum + value, count: count + 1 });
-  }, new Map());
-};
+class RatingsSummary {
+  private sum: number = 0;
+  private count: number = 0;
+  get total() {
+    return this.sum;
+  }
+  get avg() {
+    return this.count ? this.sum / this.count : 0;
+  }
+  add(rating: number) {
+    this.sum += rating;
+    this.count += 1;
+    return this;
+  }
```

Comment:
Eslint complaints about expecting a blank line between class members here. Maybe the rule could be disabled in .eslintrc.js rules:
`'lines-between-class-members': 'off',`

Reply:
In my opinion, some spacing would help with code readability. For example, one liners could be without spacing and multi-lines could be separated?

`    'lines-between-class-members': [
      'error',
      'always',
      { 'exceptAfterSingleLine': true },
    ],`
    
It would then look like


```
class RatingsSummary {
  private sum: number = 0;
  private count: number = 0;

  get total() {
    return this.sum;
  }

  get avg() {
    return this.count ? this.sum / this.count : 0;
  }

  add(rating: number) {
    this.sum += rating;
    this.count += 1;
    return this;
  }
};
```

Reply:
I added blank lines, I think that helps the readability

I just had eslint off locally, and unfortunately prettier won't add those lines